packages/client/src/clients/feed/feed.ts

```diff
@@ -624,13 +622,15 @@ class Feed {
     return result;
   }
 
-  private async makeBulkStatusUpdate(type: "seen" | "read" | "archive") {
+  private async makeBulkStatusUpdate(
+    status: BulkUpdateMessagesInChannelProperties["status"],
+  ) {
     // The base scope for the call should take into account all of the options currently
     // set on the feed, as well as being scoped for the current user. We do this so that
     // we ONLY make changes to the messages that are currently in view on this feed, and not
     // all messages that exist.
     const options = {
-      user_ids: [this.knock.userId],
+      user_ids: [this.knock.userId!],
```

Comment:
Nit: Do we need the `!` here? We should check if the types are correct since before it was defined.

Reply:
We do as the type can be undefined but it will definitely be set here